src/content/docs/telemetry-data-platform/manage-data/query-limits.mdx

```diff
@@ -204,5 +204,55 @@ Attributes on `newrelic.resourceConsumption.impact`
 - `Impact` - A count of what is happening when resource has exceeded set limit, i.e dropped requests. 
 - `consumingAccountId` - The New Relic account where the resource is being consumed.
 
- 
+## Set alerts on resource metrics
+
+While building a dashboard to see all your limits is handy, being able to automate it is even better. You can set alerts on your limit metrics to provide updates on limits changes.
+
+<Callout variant="tip">
+Because we currently only have metrics on 1 minute time windows, setting TimeWindow = 1 minute, will cover them all. Eventually, we make more metrics available, you might want to set separate alerts for limits that are enforced by different time windows.
+</Callout>
+
+You can use the following NRQL queries to create alerts. Learn about creating alerts with [NRQL queries here](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/#create).
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id=""
```

Comment:
I always add IDs; maybe it works fine without them, but I think it can be nice to control these and make sure they're simple